cu-webteam/d8-platform
# Featured content 3

## Description

Fixes the padding and margins within and around Featured Content and sidebars for mobile

## Related PRs

N/A

## Todos
 
 - [ ] Documentation (Note: all style changes. Aren't they kind of self evident?)
 
## Deploy Notes

Style changes only (CSS).

## Steps to Test or Reproduce

Check site on mobile device or in responsive design mode to verify the padding and margins of Featured Content and sidebar items

```

```
